ui/lib/kmip/addon/components/header-credentials.js

```diff
@@ -1,11 +1,13 @@
-import Component from '@ember/component';
+import Component from '@glimmer/component';
 import { inject as service } from '@ember/service';
-import layout from '../templates/components/header-credentials';
 
-export default Component.extend({
-  layout,
-  tagName: '',
-  secretMountPath: service(),
-  scope: null,
-  role: null,
-});
+export default class HeaderCredentialsComponent extends Component {
+  @service secretMountPath;
+
+  get scope() {
```

Comment:
Since we're not defaulting to anything other than what it would normally be, can we remove these getters and just use `@scope` and `@role` in the template?

Reply:
Yeah, I had the same thought. Thanks for confirming it.